opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json

```diff
@@ -480,11 +480,11 @@
         "label": "Has this data been verified with hard copy forms",
         "options": [
           {
-            "key": "Yes",
+            "key": "verified",
             "text": "Yes"
```

Comment:
The key and the test should be similar. The way it currently is leads to confusion

Reply:
this was dones to help with saving the saving of values "verified" and "not verified" which is was they want to see on filled forms